airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/gradle.py

```diff
@@ -87,7 +87,7 @@ async def _run(self, *args: Any, **kwargs: Any) -> StepResult:
             "gradle",
             "gradlew",
             "settings.gradle",
-            "build.gradle",
```

Comment:
is this intentionally removing the build.gradle entry?

Reply:
It seems like this particular change belongs in a different, later commit.

Reply:
@edgao It is in there twice for some reason, so I just co-opted the second one to match the Kotlin variant.

Reply:
@aaronsteers done